cub/cub/thread/thread_operators.cuh

```diff
@@ -585,7 +505,7 @@ struct CubOperatorToDpx<cub::Sum, T>
 // };
 
 template <typename ReduceOp, typename T>
-using cub_operator_to_dpx_t = CubOperatorToDpx<ReduceOp, T>;
+using cub_operator_to_dpx_t = typename CubOperatorToDpx<ReduceOp, T>::type;
```

Comment:
This changes behavior, but the previous version really looked like a bug to me. @fbusato, does this look good to you?

Reply:
you are right. Your version is the right one